find_similar/core.py

```diff
@@ -20,10 +23,12 @@ def find_similar(
     :param dictionary: default = None. If you want to replace one words to others you can send the dictionary.
     :return: Result list sorted by similarity percent
     """
+    stop_words_summary = add_nltk_stopwords(STOP_WORDS_NO_LANGUAGE, language)
+
     if isinstance(text_to_check, TokenText):
         text_to_check_tokens = text_to_check.tokens
     else:
-        text_to_check_tokens = tokenize(text_to_check, STOP_WORDS, dictionary)
+        text_to_check_tokens = tokenize(text_to_check, stop_words_summary, dictionary)
```

Comment:
I am thinking about this parameter. We import `STOP_WORDS_NO_LANGUAGE, tokenize, add_nltk_stopwords` to create stop_words_here and then send its to tokenize. Me be better way is just make `language` parameter in `tokenize` function and get stop words inside? What do you think about it?

Reply:
Thought about it but decided to leave def tokenize as is, so that it does not depend on nltk. May be create a class for stop_words and import only one class instead of function and constant?

Reply:
Okay. But now we use `word_tokenize` it's depend on nltk `from nltk.tokenize import word_tokenize`. But later we can send this function as param and will break this dependence.  

"May be create a class for stop_words and import only one class instead of function and constant?" - yes may be or high level function

Reply:
Done with language parameter in tokenize